### PR TITLE
SEO-175680-WPF-UG-page-with-redirects-links

### PR DESCRIPTION
**Pull request by Edithabich**
Hi @MallikaRK ,
Have made changes on the redirect links. Please review.
Thanks.

Regards,
Edith.

### CODE REVIEW

File: wpf/DataGrid/Column-Types.md [hunk 1]
```diff
@@ -1951,7 +1951,7 @@ this.dataGrid.Columns.Add(new GridHyperlinkColumn() { HeaderText = "Country Link
 
 You can allow end-user to navigate the `Uri` when the cell value contains valid `Uri` address or using [CurrentCellRequestNavigate](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.SfDataGrid.html) event. The `CurrentCellRequestNavigate` occurs when the current cell in `GridHyperlinkColumn` is clicked for navigation.
```
Comment (Yvone-Atieno) [inline wpf/DataGrid/Column-Types.md:1952]: Update the correct cr link

File: wpf/DataGrid/Editing.md [hunk 2]
```diff
@@ -283,7 +283,7 @@ SfDataGrid triggers the following events during editing.
 
 [CurrentCellBeginEdit](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.SfDataGrid.html) event occurs when the [CurrentCell](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.GridCurrentCellManager.html#Syncfusion_UI_Xaml_Grid_GridCurrentCellManager_CurrentCell) enter into edit mode. [CurrentCellBeginEditEventArgs](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.CurrentCellBeginEditEventArgs.html) has following members which provides information for `CurrentCellBeginEdit` event.
```
Comment (Yvone-Atieno) [inline wpf/DataGrid/Editing.md:284]: Update correct cr link
image

File: wpf/Multi-Column-DropDown/Popup-customization.md [hunk 3]
```diff
@@ -91,7 +91,7 @@ SfMultiColumnDropDownControl provides the following events for popup customizati
  
 ### PopupClosing event
 
-[PopupClosing](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.SfMultiColumnDropDownControl.html) event is fired when the popup is closing. You can use this event to skip the popup closing and [PopupClosingEventArgs](http://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.PopupClosingEventArgs.html) provides data for `PopupClosing` event. You can skip the popup closing by setting `args.Cancel` as true.
+[PopupClosing](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.SfMultiColumnDropDownControl.html) event is fired when the popup is closing. You can use this event to skip the popup closing and [PopupClosingEventArgs](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.PopupClosingEventArgs.html) provides data for `PopupClosing` event. You can skip the popup closing by setting `args.Cancel` as true.
```
Comment (Yvone-Atieno) [inline wpf/Multi-Column-DropDown/Popup-customization.md:94]: Check all cr links and update the correct links

File: wpf/Multi-Column-DropDown/Popup-customization.md [hunk 4]
```diff
@@ -105,7 +105,7 @@ void sfMultiColumn_PopupClosing(object sender, PopupClosingEventArgs args)
 
 ### PopupClosed event
 
-[PopupClosed](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.SfMultiColumnDropDownControl.html) event is fired when the popup is closed. [PopupClosedEventArgs](http://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.PopupClosedEventArgs.html) provides data for `PopupClosed` event
+[PopupClosed](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.SfMultiColumnDropDownControl.html) event is fired when the popup is closed. [PopupClosedEventArgs](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.PopupClosedEventArgs.html) provides data for `PopupClosed` event
```
Comment (Yvone-Atieno) [inline wpf/Multi-Column-DropDown/Popup-customization.md:108]: Change cr link

File: wpf/Multi-Column-DropDown/Popup-customization.md [hunk 5]
```diff
@@ -119,7 +119,7 @@ void sfMultiColumn_PopupClosed(object sender, PopupClosedEventArgs args)
 
 ### PopupOpening event
 
-[PopupOpening](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.SfMultiColumnDropDownControl.html) event is fired when the popup is opening. You can use this event to skip the popup opening and [PopupOpeningEventArgs](http://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.PopupOpeningEventArgs.html) provides data for `PopupOpening` event. You can skip the popup opening by setting `args.Cancel` as true.
+[PopupOpening](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.SfMultiColumnDropDownControl.html) event is fired when the popup is opening. You can use this event to skip the popup opening and [PopupOpeningEventArgs](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.PopupOpeningEventArgs.html) provides data for `PopupOpening` event. You can skip the popup opening by setting `args.Cancel` as true.
```
Comment (Yvone-Atieno) [inline wpf/Multi-Column-DropDown/Popup-customization.md:122]: Update correct cr links

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: wpf/DataGrid/Editing.md (hunk 2 above)
Comment (Edithabich) [reply wpf/DataGrid/Editing.md:284]: Resolved

File: wpf/Multi-Column-DropDown/Popup-customization.md (hunk 3 above)
Comment (Edithabich) [reply wpf/Multi-Column-DropDown/Popup-customization.md:94]: rechecked and updated proper links

File: wpf/Multi-Column-DropDown/Popup-customization.md (hunk 4 above)
Comment (Edithabich) [reply wpf/Multi-Column-DropDown/Popup-customization.md:108]: resolved

File: wpf/Multi-Column-DropDown/Popup-customization.md (hunk 5 above)
Comment (Edithabich) [reply wpf/Multi-Column-DropDown/Popup-customization.md:122]: updated

File: wpf/DataGrid/Column-Types.md [hunk 6]
```diff
@@ -1547,7 +1547,7 @@ this.dataGrid.Columns.Add(new GridComboBoxColumn() { HeaderText = "Product Name"
 {% endhighlight %}
 {% endtabs %}
 
-SfDataGrid triggers, [CurrentCellDropDownSelectionChanged](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.SfDataGrid.html) event, when the SelectedValue is changed.[CurrentCellDropDownSelectionChangedEventArgs](http://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.CurrentCellDropDownSelectionChangedEventArgs.html) of `CurrentCellDropDownSelectionChanged` event provides the information about the changed cell value.
+SfDataGrid triggers, [CurrentCellDropDownSelectionChanged](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.SfDataGrid.html) event, when the SelectedValue is changed.[CurrentCellDropDownSelectionChangedEventArgs](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.CurrentCellDropDownSelectionChangedEventArgs.html) of `CurrentCellDropDownSelectionChanged` event provides the information about the changed cell value.
```
Comment (Yvone-Atieno) [inline wpf/DataGrid/Column-Types.md:1550]: Check this link
image

Comment (Edithabich) [reply wpf/DataGrid/Column-Types.md:1550]: resolved

File: wpf/DataGrid/Column-Types.md [hunk 7]
```diff
@@ -1789,7 +1789,7 @@ this.dataGrid.Columns.Add(new GridMultiColumnDropDownList() { ItemsSource = view
...
```
[the remark this answers is not in the document]
Comment (Edithabich) [reply wpf/DataGrid/Column-Types.md:1792]: resolved

File: wpf/DataGrid/Column-Types.md [hunk 8]
```diff
@@ -2758,7 +2758,7 @@ public class GridCellTextBoxRendererExt:GridCellTextBoxRenderer
 
 ### Create the renderer of existing column
 
-You can change the renderer of existing column by removing the predefined cell type value from [CellRenderers](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.SfDataGrid.html#Syncfusion_UI_Xaml_Grid_SfDataGrid_CellRenderers) collection and add the newly derived renderer from [GridVirtualizingCellRenderer](http://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.Cells.GridVirtualizingCellRenderer%602.html).
+You can change the renderer of existing column by removing the predefined cell type value from [CellRenderers](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.SfDataGrid.html#Syncfusion_UI_Xaml_Grid_SfDataGrid_CellRenderers) collection and add the newly derived renderer from [GridVirtualizingCellRenderer](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.Cells.GridVirtualizingCellRenderer%602.html).
```
Comment (Yvone-Atieno) [inline wpf/DataGrid/Column-Types.md:2761]: Check the last link

File: wpf/DataGrid/Column-Types.md [hunk 9]
```diff
@@ -1306,15 +1306,15 @@ this.dataGrid.Columns.Add(new GridCheckBoxColumn() { HeaderText = "Is Delivered"
 
 `GridCheckBoxColumn` allows you to customize check box state and its alignment.
 
-* [IsThreeState](http://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.GridCheckBoxColumn.html) - By default, the `GridCheckBoxColumn` has `Checked` and `Unchecked` state. You can enable another `Intermediate` state setting `IsThreeState` property to `true`.
+* [IsThreeState](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.GridCheckBoxColumn.html) - By default, the `GridCheckBoxColumn` has `Checked` and `Unchecked` state. You can enable another `Intermediate` state setting `IsThreeState` property to `true`.
```
Comment (Yvone-Atieno) [inline wpf/DataGrid/Column-Types.md:1309]: Check this link

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: wpf/DataGrid/Column-Types.md (hunk 9 above)
Comment (Edithabich) [reply wpf/DataGrid/Column-Types.md:1309]: updated

File: wpf/DataGrid/Column-Types.md (hunk 8 above)
Comment (Edithabich) [reply wpf/DataGrid/Column-Types.md:2761]: updated

**Review by Aishwarya-Ganesan — APPROVED** [review]
Fine.

**Review by Aishwarya-Ganesan — APPROVED** [review]
Fine.

**Pull request merged**